### ISSUE TITLE
May I change the UI language?

### ISSUE THREAD

**Issue by yichuang-zh** [jliljebl/flowblade-forum#29]

**Comment by jliljebl** [comment]
Yes its application menu *Edit->Preferences* and in *View* tab there is drop down menu *Force language*, select *English.*

**Comment by yichuang-zh** [comment]
Got it~
Thank you~

**Comment by jesanchezg** [comment]
This is not possible in Flowblade version 1.12.

![NoPossibleChangeLanguageFlowblade](https://user-images.githubusercontent.com/28114749/84082073-3a410f00-a9ad-11ea-983d-f7384069ef20.png)
